system/jlib/jexcept.cpp

```diff
@@ -1101,6 +1101,12 @@ static void throwSigSegV()
 }
 #endif
 
+static std::atomic<unsigned> jobAborted { 0 };
+void jlib_decl setJobAborted(unsigned _abortVal)
```

Comment:
minor: given this is in jlib and not necessarily in a processes that are running 'jobs'.
perhaps setProcessAborted would be better?

Reply:
ok, I will change the name.